src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt

```diff
@@ -33,35 +61,37 @@ object ParserSetup {
      * changed to "executable".
      */
     private fun makeFileExecutable(targetFile: File) {
-        LOG.info("Making parser file executable")
+        logger.info("Making parser file executable")
         val command = arrayOf("chmod", "+x", targetFile.absolutePath)
         val builder = ProcessBuilder(*command)
         builder.directory(targetFile.parentFile)
         builder.start()
     }
 
+    private fun getTmpPath(): String {
+        val tmpPath = System.getProperty("java.io.tmpdir")
+        return tmpPath.removeSuffix("/")
+    }
+
     /**
      * Checks if parser file is in the target place and it is executable.
      * if not - makes it so.
      */
     fun checkSetup() {
-        LOG.info("Checking correctness of a parser setup")
-        val pythonparserFile = File(javaClass.getResource("$PARSER_NAME.py").path)
-        val targetFile = File(TARGET_PATH)
+        logger.info("Checking correctness of a parser setup")
+        unzipParserRepo()
```

Comment:
The default value of `toUpdate` parameter is true, so do you really want to update it every time checking the setup? It's called every time tests run

Reply:
Sorry, It is my mistake, of course I don't